libvirt-wrapper.py

```diff
@@ -0,0 +1,91 @@
+import libvirt
```

Comment:
Import order doesn't confirm to PEP-8 (https://www.python.org/dev/peps/pep-0008/#imports)